resources/startup.sh

```diff
@@ -104,6 +104,15 @@ echo "importing HTTP/S proxy settings from registry"
 NEXUS_PASSWORD="${ADMINPW}" \
   nexus-scripting execute --file-payload "${NEXUS_WORKDIR}/resources/nexusConfParameters.json" "${NEXUS_WORKDIR}/resources/proxyConfiguration.groovy"
 
+echo "create cleanup policies"
```

Comment:
Perhaps `Apply` or `Execute` would be a better fit if we apply changed configuration values again?

- [ ] to be done or not to be done, it depends on our solution

Reply:
`Apply` seems to be a good fit! 👍🏻